conf/gulp-tasks/library.gulpfile.js

```diff
@@ -1,46 +1,86 @@
 const { series, parallel, src, dest, watch } = require('gulp');
-
-const rename = require('gulp-rename');
-const sass = require('gulp-sass');
+const path = require('path');
 const postcss = require('gulp-postcss');
-const uglify = require('gulp-uglify-es').default;
+const sass = require('gulp-sass');
 
 const getLibraryVersion = require('./utils/libversion');
 const { BundleType, BundleTaskFactory } = require('./bundle/bundletaskfactory');
+const { DEV_LOCALE, BUILD_LOCALES } = require('../i18n/constants');
+const LocalFileParser = require('../i18n/localfileparser');
+const TranslationResolver = require('../i18n/translationresolver');
+const Translator = require('../i18n/translator');
 
-const bundleTaskFactory = new BundleTaskFactory(getLibraryVersion());
-
-function bundle () {
-  return bundleTaskFactory.create(BundleType.MODERN)();
-}
+/**
+ * Creates the un-minified legacy JS bundle, compiles CSS, and kicks off the watch task.
+ * @returns {Promise<Function>}
+ */
+exports.dev = function devJSBundle () {
+  return createTaskFactory(DEV_LOCALE).then(devTaskFactory => {
+    return new Promise(resolve => {
+      return parallel(
+        series(devTaskFactory.create(BundleType.LEGACY_IIFE), watchJS(devTaskFactory)),
```

Comment:
Tried this out locally because it felt a little weird that the resolve here wasn't being used. here I think if you return parallel(........)(resolve) instead of just parallel(............) and have watchJS return a function or just add a function wrapper around it, I think things will work? here `watchJS(devTaskFactory)` is a stream not a function, which I believe is why gulp starts crying about `Task never defined: [object Object]`

Reply:
Ah thanks!! That did it!